when-shit-hits-the-fan.adoc

```diff
@@ -597,19 +596,19 @@ more complete way.
 ===== After discovery
 
 OpenSSL released new versions of their software with patches that, if
-they had been used in Bitcoin from the beginning, would also have
-solved the issue. However, using that new version of OpenSSL only in a
+used in Bitcoin since the beginning, would have solved the
+issue. However, using any new version of OpenSSL only in a
 new release of Bitcoin Core would make matters worse. Gregory Maxwell
-https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-January/007097.html[explains
-this] in another email thread in January 2015:
+explains this in another
+https://lists.linuxfoundation.org/pipermail/bitcoin-dev/2015-January/007097.html[email thread] in January 2015:
 
 [quote,Gregory Maxwell on OpenSSL upgrade,Bitcoin-dev mailing list]
 ____
 While for most applications it is generally acceptable to eagerly
 reject some signatures, Bitcoin is a consensus system where all
 participants must generally agree on the exact validity or
 invalidity of the input data.  In a sense, consistency is more
-important than "correctness".
+important than "`correctness`".
```

Comment:
He actually used straight quotes in this quote, so we should stick to that here.